doc/developers/maintainer.rst

```diff
@@ -81,10 +141,31 @@ Making a release
 
        $ python setup.py sdist
 
+   - You can also test a binary dist build using::
+
+       $ python setup.py bdist_wheel
+
+   - You can test if PyPi is going to accept the package using::
+
+       $ twine check dist/*
```

Comment:
Not sure about the point of building the wheel locally and running `twine check` on it. I think using only wheels built in CI is less prone to confusion.

Reply:
We still have to build the `sdist` locally anyway. So we should check that one at least locally. Also, the build CI is pretty slow, and doing a check before triggering that is not a bad idea.